DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests/AdminFieldsDataServiceTests.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.DataServices.UserDataServiceTests
+{
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public partial class UserDataServiceTests
+    {
+        [Test]
+        public void GetAnswerCountForCourseAdminField_returns_expected_count()
```

Comment:
Should we also add a test for deletion?

Reply:
Created a test DeleteAllAnswersForCourseAdminField_deletes_all_answers